src/Handler/Table/Preview/PreviewTableHandler.php

```diff
@@ -90,10 +90,17 @@ public function __invoke(
                 assert(is_numeric($command->getChangeUntil()), 'PreviewTableCommand.changeUntil must be numeric timestamp');
             }
 
-            if ($command->hasOrderBy() && $command->getOrderBy()) {
-                /** @var PreviewTableOrderBy $orderBy */
-                $orderBy = $command->getOrderBy();
-                assert($orderBy->getColumnName() !== '', 'PreviewTableCommand.orderBy.columnName is required');
+            if ($command->getOrderBy()->count()) {
```

Comment:
je tady potreba ten if ? getOrderBy() je `array|RepeatedField` takze se to pripadne neiteruje

Reply:
fixnuto